scripts/vet.sh

```diff
@@ -70,6 +70,9 @@ not git grep "\(import \|^\s*\)\"google.golang.org/grpc/interop/grpc_testing" --
 # - Ensure that no trailing spaces are found.
 not git grep '[[:blank:]]$'
 
+# - Ensure that all files have a terminating newline.
+git ls-files | not xargs -r -I {} sh -c '[ -n "$(tail -c 1 "{}" 2>/dev/null)" ] && echo "{}: No terminating new line found"' | fail_on_output
```

Comment:
I don't think you want this `-r` since there should always be results from `git ls-files`.

Reply:
Done